package com.uber.cadence;

import java.util.*;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class GetCrossClusterTasksRequest {
  private List<Integer> shardIDs = new ArrayList<>();
  ;
  private String targetCluster;
}
